internal/core/service/services.go

```diff
@@ -1 +1,156 @@
 package service
+
+import (
+	"fmt"
+
+	"github.com/consultaUrgencias/internal/core/domain"
+	"github.com/consultaUrgencias/internal/repositories/database"
+)
+
+func RequiereProcedimiento() {
+	var pacienteRequiereProcedimiento bool
+	fmt.Println("El paciente requiere un procedimiento?")
+	fmt.Scanln(&pacienteRequiereProcedimiento)
+	fmt.Printf("El paciente: %v, requiere un procedimiento", pacienteRequiereProcedimiento)
+
+	if pacienteRequiereProcedimiento {
+		var tipoDeProcedimiento string
+		fmt.Println("Cuál es el tipo de procedimiento qe requiere el paciente: inmediato, ambulatorio")
+		fmt.Scanln(&tipoDeProcedimiento)
+		fmt.Println("El paciente requiere un procedimiento", tipoDeProcedimiento)
+
+		switch tipoDeProcedimiento {
+		case "inmediato":
+			ProcedimientoInmediato()
+		case "ambulatorio":
+			ProcedimientoAmbuatorio()
+		}
+	}
+
+}
+
+func ProcedimientoInmediato() {
+	var nombreProcedimientoInmediato string
+	fmt.Println("Cuál es el procedimiento inmediato que requiere el paciente: lavado de oido, curacion, sutura, inmovilizacion")
+	fmt.Scanln(&nombreProcedimientoInmediato)
+	fmt.Println("El paciente requiere un procedimiento", nombreProcedimientoInmediato)
+
+	switch nombreProcedimientoInmediato {
+	case "lavado de oido":
+		ProcedimientoLavadoDeOido()
+	case "curacion":
+		ProcedimientoCuracion()
+	case "sutura":
+		ProcedimientoSutura()
+	case "inmovilzacion":
+		ProcedimientoInmovilizacion()
+	}
+}
+
+func ProcedimientoAmbuatorio() {
+	var ordenDeProcedimiento string
+	fmt.Println("Cuál es el procedimiento ambulatorio que requiere el paciente")
+	fmt.Scanln(&ordenDeProcedimiento)
+	fmt.Printf("El paciente requiere %s, tramitarlo en su IPS primaria de atencion", ordenDeProcedimiento)
+
+}
+
+func ProcedimientoLavadoDeOido() {
+	getProcedimientos()
+	for "lavado de oido", []domain.Examenes := range map [string]database.procedimientos {
+		fmt.Println("Los elementos que requiere para el lavado de oido son:",["lavado de oido"] domain.Examenes)
+	}
+		
```

Comment:
Esta parte me preocupa mucho, hemos revisado for y recepción de variables en muchas ocasiones y son las bases, quizás antes de continuar con la app tengamos una clase para repasarlo porque no hay avance o claridad respecto a este tema y es muy importante. te dejo como debería ser:
```suggestion
func ProcedimientoLavadoDeOido() {
	dataProcedimientos := database.GetProcedimientos()
	for key , dataProcedimiento := range dataProcedimientos {
		if key == "lavado de oido" {
			fmt.Println(dataProcedimiento)
		}
	}
	//otra solución
	lavadoDeOidos := procedimientos["lavado de oido"]
		fmt.Println(lavadoDeOidos)
```

Reply:
Creo que si es necesara la clae adicional para consolidar el manejo de for.